tests/nlu/test_evaluation.py

```diff
@@ -485,7 +495,13 @@ def test_response_evaluation_report(tmpdir_factory):
         ),
     ]
 
-    result = evaluate_response_selections(response_results, report_folder)
+    result = evaluate_response_selections(
+        response_results,
+        report_folder,
+        successes=False,
+        errors=False,
+        disable_plotting=True,
```

Comment:
Should we add a test with `disable_plotting=False` and check if the plot is actually stored?

Reply:
It is tested in the `cli` package.